package-lock.json

```diff
@@ -35,7 +35,7 @@
     "@types/cheerio": {
       "version": "0.22.2",
       "resolved": "https://registry.npmjs.org/@types/cheerio/-/cheerio-0.22.2.tgz",
-      "integrity": "sha512-7YJQbllem83umRzZ+ry6RZFAbBWNoHvbiW6tE5VEAJIoX6F22hrKrBw3ibRvHm2zPnOhZGrZJusAbp1zZKcLzQ=="
+      "integrity": "sha1-U5Ylh0vIVghq1JHC/cmxDAWuMI4="
```

Comment:
I think you use a different version of npm, so the sha algorithm is different. So I think you should remove this file from the PR, and try to upgrade your npm.